test/CMakeLists.txt

```diff
@@ -104,7 +104,10 @@ add_executable(benchmarks dag_benchmarks.cc grid_benchmarks.cc system_benchmarks
 target_link_libraries(benchmarks PRIVATE test_main)
 set_test_target_parameters(benchmarks dag_benchmarks.cc grid_benchmarks.cc system_benchmarks.cc)
 
-add_subdirectory(system)
+if(CELERITY_ENABLE_MPI)
+  add_subdirectory(system)
+endif()
```

Comment:
Should we not run this in the single-node case too?

Reply:
These are either `distr_tests` which essentially are all skipped, and `mpi_tests`, which instantiate `mpi_communicator` explicitly and don't compile without MPI. Maybe the directory should be named differently - `multi_node` or something similar.